src/test/ui/consts/const-int-arithmetic.rs

```diff
@@ -0,0 +1,116 @@
+// run-pass
+
+#![feature(const_panic)]
+#![feature(const_if_match)]
+#![feature(saturating_neg)]
+#![feature(const_checked_int_methods)]
+#![feature(const_euclidean_int_methods)]
+#![feature(const_overflowing_int_methods)]
+#![feature(const_saturating_int_methods)]
+#![feature(const_wrapping_int_methods)]
+
+use std::i8;
+
+macro_rules! suite {
+    ($( $expr:expr, $result:expr; )+) => {
+        $( const _: () = { let _ = $expr; () }; )+
```

Comment:
Would be good to compare the results of the constant vs. runtime eval rather than using `const _` here.

Reply:
This is done now.